src/nucname.h

```diff
@@ -138,13 +143,19 @@ namespace nucname
         NaNEstr += " --> ";
         NaNEstr += nucnow;
       }
-      const char* NaNEstr_rtn = NaNEstr.c_str();
-      return NaNEstr_rtn;
+    };
+
+    /// Generates an informational message for the exception
+    /// \return The error string
+    virtual const char* what() const throw() 
+    {
+      return NaNEstr.c_str();     
```

Comment:
Good catch!!! This was probably always wrong and just lucky that we haven't been impacted before now.